app/pages/project/project-navbar.jsx

```diff
@@ -0,0 +1,343 @@
+import partition from 'lodash/partition';
+import SOCIAL_ICONS from '../../lib/social-icons';
+import Thumbnail from '../../components/thumbnail';
+import isAdmin from '../../lib/is-admin';
+
+// constants
+const AVATAR_SIZE = 100;
+
+function HomeTab({ project, projectPath, translation, onClick, children }) {
+  const avatarClasses = classnames('tabbed-content-tab', {
+    '`beta`-approved': project
+  });
+  return (
+    project.redirect ?
+    <a
+      href={project.redirect}
+      className="tabbed-content-tab"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Visit {translation.display_name}
+    </a> :
+    <IndexLink
+      to={projectPath}
+      activeClassName="active"
+      className={avatarClasses}
+      onClick={onClick}
+    >
+      {children}
+    </IndexLink>
+  );
+}
+
+function AboutTab({ project, projectPath, onClick}) {
+  if (!project.redirect) {
+    return (
+      <Link
+        to={`${projectPath}/about`}
+        activeClassName="active"
+        className="tabbed-content-tab"
+        onClick={onClick}
+      >
+        <Translate content="project.nav.about" />
+      </Link>
+    );
+  }
+};
+
+function RecentsTab({ projectPath, user }) {
+  return (
+    user ?
+    <Link
+      to={`${projectPath}/recents`}
+      activeClassName="active"
+      className="tabbed-content-tab"
+    >
+      <Translate content="project.nav.recents" />
+    </Link> :
+    null
+  );
+};
+
+function AdminTab({ project }) {
+  const adminPath = `/admin/project_status/${project.slug}`;
+  return (
+    (isAdmin()) ?
+      <Link
+        to={`${adminPath}/`}
+        activeClassName="active"
+        className="tabbed-content-tab"
+      >
+        <Translate content="project.nav.adminPage" />
+      </Link> :
+      null
+  );
+}
+
+function LabTab({ project, projectRoles, user }) {
+  const labPath = `/lab/${project.id}`;
+
+  function userHasLabAccess() {
+    return projectRoles.some(role => {
+      if (role.links.owner.id === user.id) {
+        return role.roles.includes('owner') || role.roles.includes('collaborator');
+      }
+    });
+  }
+
+  return (
+    (user && userHasLabAccess()) ?
+    <Link
+      to={`${labPath}/`}
+      activeClassName="active"
+      className="tabbed-content-tab"
+    >
+      <Translate content="project.nav.lab" />
+    </Link> :
+    null
+  );
+}
+
+const ProjectNavbar = ({
+  loading,
+  project,
+  projectAvatar,
+  projectRoles,
+  routes,
+  translation,
+  user,
+  workflow
+}, context) => {
+  const { path } = routes[2] ? routes[2] : '';
+  const projectPath = `/projects/${project.slug}`;
+  const activeElement = project && (path === 'collections' || path === 'favorites');
+  const collectClasses = classnames({
+    'tabbed-content-tab': true,
+    'active': activeElement
+  });
+
+  // helpers
+  const logClick = () => {
+    if (context && context.geordi) {
+      return context.geordi.makeHandler('project-menu');
+    }
+  };
+
+  const checkWorkflow = () => {
```

Comment:
This is a component. I think a better name would be `ClassifyLink` because that is what it renders.

Reply:
I see what you mean, but it's ultimately called by `ClassifyTab`.

Reply:
Sorry, I'm not following what you mean here. This is a component that's rendered by `ClassifyTab` for internal links to the classify page and the name is a bit misleading.